plugins/bulk-import/package.json

```diff
@@ -5,7 +5,7 @@
   "types": "src/index.ts",
   "license": "Apache-2.0",
   "publishConfig": {
-    "access": "public",
+    "access": "restricted",
```

Comment:
i don't think this is needed, you might want to add https://github.com/janus-idp/backstage-plugins/blob/main/plugins/argocd/package.json#L7